src/NovelRT.Interop/NrtNovelRunner.cpp

```diff
@@ -66,34 +65,6 @@ extern "C"
         return NRT_SUCCESS;
     }
 
-    NrtResult Nrt_NovelRunner_getAudioService(NrtNovelRunnerHandle runner, NrtAudioServiceHandle* outputService)
-    {
-        if (runner == nullptr)
-        {
-            Nrt_setErrMsgIsNullptrInternal();
-            return NRT_FAILURE_NULL_INSTANCE_PROVIDED;
-        }
-
-        if (outputService == nullptr)
-        {
-            Nrt_setErrMsgIsNullptrInternal();
-            return NRT_FAILURE_NULL_ARGUMENT_PROVIDED;
-        }
-
-        NovelRunner* cRunner = reinterpret_cast<NovelRunner*>(runner);
-        _audioCollection.push_back(cRunner->getAudioService());
-
-        auto ptr = _audioCollection.back().get();
-        if (ptr == nullptr)
-        {
-            return NRT_FAILURE_NULL_INSTANCE_PROVIDED;
-        }
-
-        *outputService = reinterpret_cast<NrtAudioServiceHandle>(ptr);
-
-        return NRT_SUCCESS;
-    }
-
```

Comment:
Is this part of our efforts to move away from the legacy NovelRunner?

Reply:
Yep, and is required since it won't map to anythjng in C++ land